tests/utils.py

```diff
@@ -214,9 +278,11 @@ def clean(self):
     self.flush_log()
 
     self.__temp_log_file.close()
+    self._kill_server_process()
 
-    if self.__server_process.returncode is None:
-      self.__logger.info('Server process ' + str(self.__server_process.pid) +
-          ' terminated.')
-      self.__server_process.kill()
-      self.__server_process.wait()
+
+
+  def is_process_running(self):
+    """Returns a boolean value if the server process is currently running."""
```

Comment:
nit: this comment does not tell me anything... It probably does not need to exit

Reply:
I agree.
I just felt obligated to add it when all of the other functions have docstrings.